### DISCUSSION TITLE
rtl_433 CaribouLite-ISM SoapySDR - Single Pulses only

### DISCUSSION THREAD

**Discussion by serg-piter** [merbanan/rtl_433#3016] [Q&A]
Hello, everyone

My setup is rPi Zero 2W with CaribouLite ISM HAT. SoapySDR backend. Everything built from the latest repo snapshots including rtl_433. 

The problem is that rtl_433 can't recognise any of my 433 sensors and if i'm running trace level of verbose or -A - i can see data, but rtl_433 decodes it not as "packet" but as single pulse. 

For example - 
`Detected OOK package	2024-08-01 18:36:24
Analyzing pulses...
Total count:    1,  width: 0.22 ms		(   54 S)
Pulse width distribution:
 [ 0] count:    1,  width:  216 us [216;216]	(  54 S)
Gap width distribution:
Pulse period distribution:
Pulse timing distribution:
 [ 0] count:    1,  width:  216 us [216;216]	(  54 S)
 [ 1] count:    1,  width: 10004 us [10004;10004]	(2501 S)
Level estimates [high, low]:  13342,   1468
RSSI: -1.8 dB SNR: 19.2 dB Noise: -21.0 dB
Frequency offsets [F1, F2]:     204,      0	(+0.8 kHz, +0.0 kHz)
Guessing modulation: Single pulse detected. Probably Frequency Shift Keying or just noise...
view at https://triq.org/pdv/#AAB10200D827148155`

I've tried different bandwidth and samplerates configurations but i have no ideas where to dig into next. 

Any suggestions?

**Comment by zuckschwerdt** [comment]
433M defaults to the `-Y classic` demod, try the FSK optimized `-Y minmax` demod.
